use/web/admin-panel.md

```diff
@@ -69,6 +69,46 @@ Once the form and media are successfully uploaded, click on the add button to su
 
 ![media](../../../media/a11.png)
 
+#### Monitoring
```

Comment:
Please explain in brief about that the monitoring services are for, what are the different features along with the different sections.